src/LiveComponent/doc/index.rst

```diff
@@ -3775,6 +3778,55 @@ uses Symfony's test client to render and make requests to your components::
     The ``InteractsWithLiveComponents`` trait can only be used in tests that extend
     ``Symfony\Bundle\FrameworkBundle\Test\KernelTestCase``.
 
+Test Live Collection Type
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To test the submission of a form within a Live Component (with the above `submitForm` helper) containing a `LiveCollectionType`, you first need to physically add the number of entries that you want to test in your component (as if you would click on the "Add" button on the screen).
+
+So, if the following are the forms used::
+
+    // Parent FormType used in the Live Component
+    use Symfony\UX\LiveComponent\Form\Type\LiveCollectionType;
+
+    class LiveCollectionFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options): void
+        {
+            $builder->add('children', LiveCollectionType::class, [
+                    'entry_type' => ChildFormType::class,
+                ])
+            ;
+        }
+    }
+
+    // Child Form Type used in the parent form type
+    class ChildFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options): void
+        {
+            $builder
+                ->add('name', TextType::class)
+                ->add('age', IntegerType::class)
+            ;
+        }
+    }
+
+The test of the submission of such form would be with the TestHelper::
+
+    // calling the addCollectionItem from the LiveCollectionTrait with the name of field in the form
+    $component->call('addCollectionItem', ['name' => 'children']);
+    $component->call('addCollectionItem', ['name' => 'children']);
+    //... can be called as many times as you need entries in your 'children' field
+
+    // and then submit the form
```

Comment:
```suggestion
    // ... then submit the form by providing data for all the fields in the ChildFormType:
```

Reply:
Looking at the next review, I would add at the end of your updated comment `for each added entry` or something of sort